src/sentry/api/endpoints/organization_details.py

```diff
@@ -528,6 +531,7 @@ def save(self):
                 "require_2fa": org.flags.require_2fa.is_set,
                 "codecov_access": org.flags.codecov_access.is_set,
                 "disable_member_project_creation": org.flags.disable_member_project_creation.is_set,
+                "disable_superuser_access": org.flags.disable_superuser_access.is_set,
```

Comment:
i've had trouble with options using the negative before... i would suggest calling this `allow_superuser_access`

Reply:
This is a negative since the default is False and we do not want to have data secrecy enabled by default.
If i make this allow, I will need to write a script to make everyone's allow_data_secrecy True by default and we would have to do this for every new org creation.

Reply:
What about `prevent_superuser_access`? That would avoid needing to use negation 🤷